demo/monitoring_all_process_instances/index.js

```diff
@@ -0,0 +1,89 @@
+/*function removeAllOverlays() {
+    // Remove all overlays
+    // Shape
+    bpmnElementsRegistry.removeAllOverlays('Activity_1potg3p');
+
+    // Edge
+    bpmnElementsRegistry.removeAllOverlays('Flow_1wkfbb0');
+}*/
+
+
+
+/*var style = {
+    font: {
+        color: fontColorElt.value,
+        size: fontSizeElt.value,
+    },
+    fill: {
+        color: fillColorElt.value,
+    },
+    stroke: {
+        color: strokeColorElt.value,
+    }
+};*/
+
+
+// Initialize BpmnVisualization for Time Data
+const timeBpmnVisualization = new bpmnvisu.BpmnVisualization({container: 'time-bpmn-container', navigation: {enabled: true}});
+const timeBpmnElementsRegistry = timeBpmnVisualization.bpmnElementsRegistry;
+
+// Load BPMN diagram
+timeBpmnVisualization.load(getHardwareRetailerDiagram(), {fit: {type: 'Center', margin: 30}});
```

Comment:
ℹ️ we may introduce a constant for the `loadOptions` as it is reused somewhere else in this file. This will also show that we intentionally want to use the same options for both scenarios.
Or a load function taking a BpmnVisualization parameter as we load the same BPMN diagram